src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+body {
+  font-family: Avenir, sans-serif;
```

Comment:
It's better to specify a fallback font in case the main one doesn't load. You can add a generic font family such as 'sans-serif' to the end of your font stack. Also, be sure to include 'Avenir-Book' and 'Avenir-Heavy' in your font stack if you're using them elsewhere in your CSS.